cypress/integration/login.js

```diff
@@ -7,18 +7,22 @@ describe('Login functionality', function() {
         cy.get('.Select.not-valid').should('be.visible')
     })
 
-    it('Should be able to login with role User', function () {
+    it('Should be able to login with role team lead', function () {
         cy.get('[id="loginForm.userId"]').click({force:true})
         cy.get('[aria-label="Demo User"]').click()
         cy.get('[id="loginForm.role"]').click({force:true})
-        cy.get('[aria-label="User"]').click()
+        cy.get('[aria-label="Team Lead"]').click()
         cy.get('[type="submit"]').click()
 
         cy.url().should('include', '/time-logging')
         cy.get('.page__title').contains('Timesheets')
         cy.get('.calendar').should('be.visible')
         cy.get('.tile.form').should('be.visible')
         cy.get('.user-info__title').contains('Demo User')
-        cy.get('.main-nav').find('li').should('have.length', 1)
+        cy.get('.main-nav').find('li').should('have.length', 2)
+
+	it('Should verify todays date')
```

Comment:
this it is kind of unnecessary, it does not follow syntax of new test case

Reply:
removed